### ISSUE TITLE
U2F transaction does not work on Windows 10 1903

### ISSUE THREAD

**Issue by faith0811** [cosmos/ledger-cosmos-js#13]
On Windows 10 1903(latest major update), the U2F has a new popup interface, which breaks the communication from Chrome to Ledger.
The silence interactions(such as the get app info api) perform normal. But the sign transaction api freezes after the confirm operation.

Related issue:
https://github.com/LedgerHQ/ledgerjs/issues/370
Same problem with different crypto.

I've read the manual posted before:
https://www.ledger.com/your-ledger-device-and-the-upcoming-windows-10-update/

It says that user should reinstall the crypto apps. But it does not work for me and I think it might need to do some tweak in the app itself to make it work.

**Comment by jleni** [comment]
yes, correct. Windows 10 v1903 broke U2F communication with most Ledger devices/apps.

We've been working closely with Ledger with respect to this problem. While we have improved the app v1.6.0 based on some feedback we got from Ledger, this has not completely solved the problem in all devices that run Windows.

I will discuss a bit more with Ledger to see if we can release 1.6.0 soon, but be aware that it does not solve the issue completely. 

The current consensus is that a real fix will require changes in Windows. For instance, even with changes in the apps, the pesky popups will always appear and user experience is strongly affected.

The other alternative is abandoning U2F and move directly to WebUSB, which the app already supports.
